src/components/OfferCard/OfferCard.js

```diff
@@ -32,8 +32,7 @@ const SubscriptionCard = ({
   paymentMethod,
   t
 }) => {
-  const isSubscription =
-    getData('CLEENG_OFFER_TYPE') === 'S' || isSubscriptionOffer;
+  const isSubscription = offerType === 'S' || isSubscriptionOffer;
```

Comment:
do we need `isSubscriptionOffer`? Maybe we could base only on offerType?

Reply:
`isSubscriptionOffer` was removed from the component